examples/1_simple_runbooks/configure.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+"""
+In this example we write a CLI tool with brigade and click to deploy configuration.
```

Comment:
I assume this comment is wrong.

Reply:
yeah, examples are not finished and they have undergone many iterations...